mutatest/cli.py

```diff
@@ -691,7 +722,9 @@ def main(args: argparse.Namespace) -> None:
         break_on_survival=run_mode.break_on_survival,
         break_on_error=run_mode.break_on_error,
         break_on_unknown=run_mode.break_on_unknown,
+        break_on_timeout=run_mode.break_on_timeout,
         ignore_coverage=args.nocov,
+        max_runtime=args.timeout_factor * clean_runtime_1.seconds,
```

Comment:
Heads up in my local testing, I changed this to `clean_runtime_1.total_seconds()` instead of `clean_runtime_1.seconds`. If the tests are under 1s duration the `seconds` property is set to 0.